coordinape/coordinape
# migrating updateCircle backend and frontend to hasura

Fixes #438 and Fixes #632 by migrating updateCircle backend and frontend to hasura. 

Test plan:

1. Use the Admin Circle page to change all the various settings on a circle. Verify that graphql is used in the network tab rather than laravel. 
2. Enable vouching, then nominate some users
3. Edit the circle and disable vouching
4. Check in the DB or the UI that the nominees have ended. 

Also improves zod error handling to include the field that is bad. Try setting an invalid url for the discord webhook to see this.
